### DISCUSSION TITLE
Input file representation after passing CLI args

### DISCUSSION THREAD

**Discussion by jessecarterMOOSE** [idaholab/moose#23128] [Q&A General] [answered]
If I were to pass input file modifications on the command line, like "Kernels/label/value=foo", is there a way to get a representation of what the input file would like with that modification? This would be for preserving inputs to runs that are procedurally generated.

**Comment by GiudGiud** [comment, marked as answer]
Hello

If you are using exodus output the inputs are preserved there.

Otherwise this worked natually for me, the `--show-input` being the command line option needed

./moose_test-opt -i tests/kernels/simple_diffusion/simple_diffusion.i --show-input AuxVariables/a/type=MooseVariableFVReal

Guillaume
